src/libraries/Common/src/Interop/Unix/System.Native/Interop.Fcntl.cs

```diff
@@ -17,6 +17,9 @@ internal static partial class Fcntl
             [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_FcntlSetIsNonBlocking", SetLastError=true)]
             internal static extern int SetIsNonBlocking(SafeHandle fd, int isNonBlocking);
 
+            [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_FcntlGetIsNonBlocking", SetLastError = true)]
+            internal static extern int GetIsNonBlocking(SafeHandle fd, out bool isNonBlocking);
```

Comment:
I wish we did not start with the negative naming ;( GetIsBlocking would be much easier to read IMHO.

Reply:
We already have the existing SystemNative_FcntlSetIsNonBlocking, which is just surfacing the platform's O_NONBLOCK, so it was POSIX that started the negative naming :)